backend/runner/runner.go

```diff
@@ -128,11 +129,14 @@ func (s *Service) Deploy(ctx context.Context, req *connect.Request[ftlv1.DeployR
 		return nil, connect.NewError(connect.CodeUnavailable, errors.Wrap(err, "failed to register runner"))
 	}
 
-	id, err := model.ParseDeploymentKey(req.Msg.DeploymentKey)
+	key, err := model.ParseDeploymentKey(req.Msg.DeploymentKey)
 	if err != nil {
 		return nil, connect.NewError(connect.CodeInvalidArgument, errors.Wrap(err, "invalid deployment key"))
 	}
 
+	sink := newDeploymentLogsSink(ctx, key, s.key)
+	logger := log.New(log.Default, sink)
```

Comment:
So here I was creating a new logger that would also log to the current logger via ctx, but since `Sub` isn't called on the original logger, we're missing the attributes.

Reply:
I think I might need to create a `Tee` sink, but at this point I don't have access to the original `sink` from the logger. So I might need to either expose that somehow or come up with another approach.

Reply:
I can't 100% remember, but I don't think I was suggesting creating a completely new logger was I? I thought we talked about having a `Logger.AddSink()` that would use `TeeSink` internally?